### ISSUE TITLE
Introduction notes-Gokul

### ISSUE THREAD

**Issue by payalas1550-cyber** [payalas1550-cyber/DanforthMicroInternship#3]

**Comment by Gokubala** [comment]
3. Data Products:

Plot level indices raster → plot level indices table

→ Data products sent to cloud for analysis

Geospatial Concepts

(Fundamentals of geo data, geographic information systems and their applications)



Data Preparation

(How to clean and process data for analysis)



Modeling and Predictions

(Machine learning to implement regression models to predict plant traits (yield, protein content, etc.))



Tool Proficiency

(Coding experience using Google Colab and Python)

Overview:
Creating maps, creating heatmaps using QGIS and Python, and visualizing sorghum field trial data

Predictive model plant traits using data from drones, field instruments, technicians, and climate records to predict yield, protein content, or chlorophyll fluorescence

Log, track, and visualize various metrics (hyperparameter tuning, preprocessing steps, feature importance)

Work with:
Wireless sensor data

Drone data

Climate data

Field technician observations

Crop data

Machine Learning (Preimplemented Regression Models):
Partial Least Squares Regression (PLS)

Lasso regression

Elastic Net Regression

Bayesian Ridge Regression

Extreme Gradient Boosting (XGBoost)

Support Vector Regression (SVR)

Questions for them…

Create heatmaps using (QGIS) → geospatial data software + Python

Visualize real sorghum research field trial data

Machine learning & data science

How to build a predictive model for important plant traits

(Yield, seed protein content, and chlorophyll fluorescence) — using drones, technicians, climate records, and field instruments.

Learn to track metrics

(Preprocessing steps, hyperparameter tuning, and feature importance using TensorBoard)

Goal:


Understanding plant performance

Phenode: data collector (wireless sensor network)

Collects atmospheric data (windspeed, precipitation, direction, and temperature)

Soil moisture, salinity, electrical conductivity (communicate up to 1 km away)

Field Doc:

Works in harmony with things like the Phenode to provide scientists with good data.

Improves crop allocation

Reduces need for water, fertilizer, and energy

Tool to operate with zero-carbon footprint

Data Pipeline:


1. Data Collection:

UAV data collection → multispectral images → image data transferred to edge computer in field doc



2. Data Processing:

Orthomosaic Generation (ODM) → orthomosaic image

→ image processing at the edge
